### DISCUSSION TITLE
Thank You Rspress Team!

### DISCUSSION THREAD

**Discussion by trevor-moore-dev** [web-infra-dev/rspress#1678] [General]
Wanted to post this to voice my excitement and support for Rspress.. my team and I are going to move to Rspress from Docusaurus in 2025 and we're looking forward to experiencing the improved build times both on our local boxes and on our CI server. We setup a PoC with the current amount of MD/X files we have (it's an ever-growing amount) and that was all we needed to convince us!

That said, I wanted to say thank you for all the great work you're doing, and I'm looking forward to what comes in 2025!

**Comment by Timeless0911** [comment]
Thanks for the recognition! We're excited to launch Rspress 2.0 in 2025 and hope to bring you a significantly improved experience.
